recipes/r-scevan/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: r-scevan
+  version: 1.0.1
+
+source:
+  url: https://github.com/AntonioDeFalco/SCEVAN/archive/refs/tags/v1.0.1.tar.gz  
+  sha256: 7558d31282c36418db3960b3edb9cb9815db14b0a48d8882287e11f752dfeab1       
+
+build:
+  number: 0
+  noarch: generic  # Specify that the package is noarch
+  run_exports: '{{ pin_compatible("r-scevan", max_pin="x.x") }}'
+  rpaths:
+    - lib/R/lib
+    - lib
+  script: "${R} CMD INSTALL --build . "
```

Comment:
This is now missing `miccec/yaGST` probably.

Reply:
added miccec/yaGST installation via devtools to build.sh script